src/style.css

```
@@ -0,0 +1,28 @@
+body {
+  margin: 0;
+  padding: 20px;
+  background-color: #000;
+}
+
+.article,
+.orange-box,
```

don't call classes by color

```suggestion
.inner_block,
```